gems/pending/appliance_console.rb

```diff
@@ -461,6 +462,32 @@ module ApplianceConsole
         dbhost, database, region = ApplianceConsole::Utilities.db_host_database_region
         press_any_key
 
+      when I18n.t("advanced_settings.db_replication")
+        say("#{selection}\n\n")
+
+        options = {
+          "Initialise Database Replication Master"  => "master",
```

Comment:
I like your suggestions of  using `Primary` over `Master` So I'm also going to rename `appliance_console/database_replication_master` to `appliance_console/database_replication_primary`